pom.xml

```diff
@@ -1,4 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
That's new.

Maybe, we shouldn't insert it here? It will get overwritten next time we generate it, anyway.